core/cloudflow-flink/src/main/scala/cloudflow/flink/FlinkStreamlet.scala

```diff
@@ -270,7 +269,9 @@ abstract class FlinkStreamlet extends Streamlet with Serializable {
  *  }
  * }}}
  */
-abstract class FlinkStreamletLogic(implicit context: FlinkStreamletContext) extends Serializable {
+abstract class FlinkStreamletLogic(implicit val context: FlinkStreamletContext) extends StreamletLogic[FlinkStreamletContext] {
+  override def getContext(): FlinkStreamletContext = super.getContext()
```

Comment:
Is this one necessary?

Reply:
yes.

Reply:
Ah, I see, it is for java compatibility. Ok.